bitnami/Dockerfile

```diff
@@ -153,5 +153,6 @@ RUN set -ex \
         /var/lib/apt/lists/* \
         /tmp/*               \
         /var/tmp/*
+RUN echo "shared_preload_libraries = 'timescaledb'" >/opt/bitnami/postgresql/conf/conf.d/timescaledb.conf
```

Comment:
I still see the newer fix here instead of the `sed` one?

Reply:
Sorry, seems I forgot to commit the change. It's fixed now.